mne_lsl/stream_viewer/stream_viewer.py

```diff
@@ -29,16 +23,18 @@ def __init__(self, stream_name=None):
     def start(self, bufsize=0.2):
         """Connect to the selected amplifier and plot the streamed data.
 
-        If ``stream_name`` is not provided, look for available streams on the
-        network.
-
         Parameters
         ----------
         bufsize : int | float
             Buffer/window size of the attached Viewer.
             The default ``0.2`` should work in most cases since data is fetched
             every 20 ms.
         """
+        from qtpy.QtWidgets import QApplication
```

Comment:
Rather than any of this stuff you could in theory use MNE-Python's app framework, which also has the advantage of setting a nice icon etc. like in mne-qt-browser

https://github.com/mne-tools/mne-qt-browser/blob/a267775f2d0edcf188507c4a66c566cc25182fa2/mne_qt_browser/_pg_figure.py#L44

Reply:
True, but I'll keep that for whenever I get the motivation to continue https://github.com/mne-tools/mne-lsl/pull/174 ;)